tests/tests/conftest.py

```diff
@@ -1,22 +1,13 @@
-import os
+import pathlib
 import sys
 
 import pytest
 from cellfinder_core.download.cli import main as cellfinder_download
 from imlib.general.config import get_config_obj
 
-data_dir = os.path.join(
-    os.getcwd(),
-    "tests",
-    "data",
-    "brain",
-)
-test_output_dir = os.path.join(
-    os.getcwd(),
-    "tests",
-    "data",
-    "registration_output",
-)
+test_data_dir = pathlib.Path(__file__) / ".." / ".." / "data"
```

Comment:
N.B. there's also the `parents[1]` option. But IMO this is more obvious what's going on